index.html

```diff
@@ -132,10 +132,16 @@ <h2 class="intro-date">2015.10.15(THU)~16(FRI)</h2>
     <!-- close body section -->
 
     <section id="daily-menu" class="pt-5">
+     
       <div class="container-fluid mt-5 pb-5 bg-dark py-md-5 px-5">
         <div class="row p-lg-5 bg-dark">
+          <div class="text-center centered-title mb-5 pb-lg-4">
+            <h2 class="text-light text-decoration-underline">Main Dishes</h2>
+          </div>
```

Comment:
- [OPTIONAL] You've done a great job adding a heading and styling it however let us stick to the design as much as possible, it is very important later on in our job. Please remove the white line and add a small red one similar to the design below
Your design 👇 
![Screenshot from 2022-07-28 12-31-52](https://user-images.githubusercontent.com/87965534/181473014-9b94c67a-f5fa-4497-90ec-7bf98d06030c.png)

Original design 👇 
![Screenshot from 2022-07-28 12-31-00](https://user-images.githubusercontent.com/87965534/181472833-09d6ea9c-ed6b-42e1-94dd-1e585a08fd51.png)

You can do that simply by adding this 👇 since it is already styled
```
<div class="line"></div>
```

Reply:
@Sahar-AbdelSamad Thank you for this review.